Mathlib/Analysis/SpecialFunctions/Integrability/Basic.lean

```diff
@@ -0,0 +1,283 @@
+/-
+Copyright (c) 2021 Benjamin Davidson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Benjamin Davidson
+-/
+import Mathlib.Analysis.SpecialFunctions.Log.NegMulLog
+import Mathlib.Analysis.SpecialFunctions.NonIntegrable
+import Mathlib.Analysis.SpecialFunctions.Pow.Deriv
+
+/-!
+# Integration of specific interval integrals
```

Comment:
This docstring is the docstring of the other, old, file. It does not correspond to the contents of the current file. Could you replace it with something more appropriate?

Reply:
Done.